src/js/user-profile/helpers.jsx

```diff
@@ -0,0 +1,73 @@
+import Raven from 'raven-js';
+
+import fullSchema1010ez from '../hca/config/form';
+import fullSchema1990 from '../edu-benefits/1990-rjsf/config/form';
+import fullSchema1990e from '../edu-benefits/1990e/config/form';
+import fullSchema1990n from '../edu-benefits/1990n/config/form';
+import fullSchema1995 from '../edu-benefits/1995/config/form';
+import fullSchema5490 from '../edu-benefits/5490/config/form';
+import fullSchema5495 from '../edu-benefits/5495/config/form';
+import fullSchema527EZ from '../pensions/config/form';
+import fullSchema530 from '../burials/config/form';
+
+
+export const formTitles = {
+  '21P-527EZ': 'Veterans pension benefits (21P-527EZ)',
+  '21P-530': 'burial benefits (21P-530)',
+  '1010ez': 'health care (10-10EZ)',
+  '22-1990': 'education benefits (22-1990)',
+  '22-1990E': 'education benefits (22-1990E)',
+  '22-1990N': 'education benefits (22-1990N)',
+  '22-1995': 'education benefits (22-1995)',
+  '22-5490': 'education benefits (22-5490)',
+  '22-5495': 'education benefits (22-5495)'
```

Comment:
@peggygannon can you please confirm that the titles such as can be seen in the screen shot, and which are listed here, should be lower case as with the form introduction pages?

Reply:
@cehsu Hi, Claire. Can I have more context about where these titles will be seen--will they appear in a save application message? Also, do you need me to provide content edits to the screen shots above?

Reply:
@peggygannon, these titles will be displayed in the user's profile, where they will be able to delete or resume their applications. Additional content edits would be greatly appreciated.

Reply:
@cehsu OK, thanks. Please make the form names lowercase to match the intro pages. 
Here are my suggested edits: 

Application for Veterans pension benefits (21P-527EZ)
Last saved on 8/29/2017 at [provide time?]  a.m. or p.m.
Your saved application will expire in 59 days. 

Button: Continue Your Application 

**Your saved application has expired**
Your application for Veterans pension benefits (21P-527EZ) has expired. If you want to still apply, please [start a new application].

Reply:
Thanks for the guidance, @peggygannon. Kindly refer to these updated screenshots.
<img width="363" alt="screen shot 2017-08-29 at 4 05 09 pm" src="https://user-images.githubusercontent.com/16051172/29848204-1fd25e88-8cd4-11e7-8dd0-5bb2b1cc253a.png">

<img width="762" alt="screen shot 2017-08-29 at 4 06 23 pm" src="https://user-images.githubusercontent.com/16051172/29848207-22c2d7a8-8cd4-11e7-816c-cc59c2b4c153.png">


<img width="810" alt="screen shot 2017-08-29 at 4 05 58 pm" src="https://user-images.githubusercontent.com/16051172/29848210-24e7e4e2-8cd4-11e7-90cc-5116a36e2ad9.png">

Reply:
@cehsu Thanks, can you also update this line: 
--Your saved application will expire in 59 days.
--Add period after this sentence:...Your saved application has expired.

Reply:
Thanks again for the helpful pointers @peggygannon, kindly refer to the updated screenshots:

<img width="770" alt="screen shot 2017-08-29 at 4 52 43 pm" src="https://user-images.githubusercontent.com/16051172/29849314-ca662fcc-8cda-11e7-9766-c6a0a7489547.png">


<img width="435" alt="screen shot 2017-08-29 at 4 53 04 pm" src="https://user-images.githubusercontent.com/16051172/29849315-cccdc0c2-8cda-11e7-9d06-bbf5d0b512d2.png">


<img width="714" alt="screen shot 2017-08-29 at 4 53 42 pm" src="https://user-images.githubusercontent.com/16051172/29849316-cf038b6a-8cda-11e7-8fa8-16a7f63f34c3.png">

Reply:
@cehsu LGTM!

Reply:
Good idea, changed. Kindly refer to updated screen shot below:
<img width="772" alt="screen shot 2017-08-30 at 12 14 08 pm" src="https://user-images.githubusercontent.com/16051172/29890602-0a8a6512-8d7d-11e7-9891-f8487fee3122.png">

Reply:
Ahh, that looks great! 👍